samples/samples-powershell/AddProductWithIdentityColumnIncluded/run.ps1

```diff
@@ -0,0 +1,27 @@
+using namespace System.Net
+
+# Trigger binding data passed in via param block
+param($Request)
+
+# Write to the Azure Functions log stream.
+Write-Host "PowerShell function with SQL Output Binding processed a request."
+
+# Update req_body with the body of the request
```

Comment:
Update comment. Several other functions have the same comment as well. Make sure to update or remove them.